bin/strata-cli/src/seed/password.rs

```diff
@@ -8,13 +9,33 @@ pub struct Password {
     seed_encryption_key: Option<[u8; 32]>,
 }
 
+pub enum HashVersion {
+    V0,
+}
+
+impl HashVersion {
+    const fn params(&self) -> (Algorithm, Version, Result<Params, argon2::Error>) {
+        match self {
+            HashVersion::V0 => (
+                Algorithm::Argon2id,
+                Version::V0x13,
+                Params::new(19_456, 2, 1, None),
```

Comment:
Recommend including a citation comment to the relevant [OWASP recommendation](https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#argon2id) to justify these magic numbers.

Reply:
Also recommend explicitly specifying the output length instead of relying on the `None` default behavior for the parameters.

Reply:
Good catch, done in 953d9f4bba437aa7e64df959fd48663f869037c9